policies/rubykaigi2017/anti-harrasment-policy-en.md

```diff
@@ -0,0 +1,32 @@
+# RubyKaigi Anti Harassment Policy
+
+At RubyKaigi, we want you to be happy.  We value your attendance.
+
+To achieve that, all participants are required to agree with the following anti harassment policy. Organizers will enforce this throughout the conference, party, and related social events. We expect cooperation from all participants to help ensure a safe environment for everybody.
+
+RubyKaigi is dedicated to providing a harassment-free conference experience for everyone. We do not tolerate harassment of conference participants in any form. Any participants violating these rules may be sanctioned or expelled from the conference at the discretion of the conference organizers.
+
+Harassment includes, but is not limited to:
+
+- Offensive verbal comments related to gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, religion (or lack thereof).
+- Sexual images in public spaces
+- Deliberate intimidation, stalking, following
+- Harassing photography or recording
+- Sustained disruption of talks or other events
+- Inappropriate physical contact
+- Unwelcome sexual attention
+- Advocating for, or encouraging, any of the above behaviour
+
+Participants asked to stop any harassing behavior are expected to comply immediately.
+
+Sponsors are also subject to this anti-harassment policy. In particular, sponsors should not use sexualized images, activities, or other material. Booth staff (including volunteers) should not use sexualized clothing/uniforms/costumes, or otherwise create a sexualized environment.
+
+If a participant engages in harassing behavior, the conference organizers may take any actions to keep the event a welcoming environment for all participants. This includes warning the offender or expulsion from the conference with no refund.
+
+If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact a member of conference staff immediately.  Conference staff can be identified by their name tags.
```

Comment:
Thank you.  There will be several dozens of conference staffs in the conference so actually "a member of" is intentional.